iOSChatExample/iOSChatExample/Models/ChatManager.swift

```diff
@@ -18,7 +18,9 @@ class ChatManager : ObservableObject{
 
     init() {
         networkManager = NetworkManager(httpClient: DefaultHttpClient())
+        // `accessKey`,`secretKey` and `forKey` are set to empty on purpose. This is just to initiate AWS service. DO NOT NEED TO CHANGE ANYTHING HERE.
         let credentials = AWSStaticCredentialsProvider(accessKey: "", secretKey: "")
+        // `region` is important, credentials are passed as empty as it was a mandatory parameter. 
```

Comment:
the README still tells user to update region, this is just additional comment right?